app/views/user_mailer/invitation.text.erb

```diff
@@ -1,5 +1,4 @@
 <% content_for :message do %>
-<%= t("mailer.general.greeting.introductory") %>
 
```

Comment:
I think this blank line needs to go, too, or else it might add extra whitespace.